### DISCUSSION TITLE
Qiita CLI の publish の挙動について

### DISCUSSION THREAD

**Discussion by r-hirakawa** [increments/qiita-discussions#496] [要望・アイデア・不具合]
開発チームのみなさま、Qiita CLI のβ版リリースありがとうございます。
現在、弊社の社外情報発信のために Qiita を活用しておりますが、このツールにより、記事執筆、社内レビュー＆承認、投稿までの流が GitHub 上で非常にスムーズに実現できるようになるため、非常に感激しております！

さっそく試してみて気づいた点ですが、以下のコマンドを実行したときに記事の内容にエラーがあると、エラーメッセージが出力されますが、コマンドの終了コードが `0` （正常）となっているため、GitHub Actions のワークフロー内でエラーが発生しても、ワークフローが正常終了してしまいます。

例：`npx qiita new` で生成した記事のタグを、初期値のまま修正しなかった場合
```
$ npx qiita publish <記事のベース名>
Validation error:
テスト [ 'タグを入力してください' ]
Nothing to publish

$ echo $?
0
```

ワークフローが正常終了しているが、Qiita に記事が投稿されていないという状態は、慣れていない担当者だと原因確認に時間がかかってしまうため、publish でエラーが発生した場合は GitHub Actions のワークフローも異常終了してくれると助かります。

**Comment by getty104** [comment]
@r-hirakawa 
Qiita CLIをご利用いただきありがとうございます！
また、ご不便おかけしてしまい申し訳ございません 🙇‍♂️
いただいたフィードバックは社内で確認し、対応検討させていただきます！
また何か気になるところあればご連絡いただけると幸いです。
よろしくお願いいたします！

**Reply by getty104** [reply]
@r-hirakawa 
お待たせいたしました！
こちらv0.3.0で対応いたしましたので、アップデートをお願いできますでしょうか :pray: 
引き続きQiita CLIをよろしくお願いいたします！
https://github.com/increments/qiita-cli/releases/tag/v0.3.0

**Reply by r-hirakawa** [reply]
@getty104 修正版のリリースありがとうございます😍
さっそく試してみた所、エラー時に終了コード `1` が返り、GitHub Actions のワークフローもエラー終了することが確認できました。
素早い修正対応ありがとうございました。

さすが Qiita 開発チームさん。スピード感が違いますね！
